src/mui/detail/Show.js

```diff
@@ -79,7 +79,7 @@ export class Show extends Component {
                             href={`#/${this.props.resource}`}
```

Comment:
for the "bug" on the log resource you may want to be able to pass a custom redirection link